### ISSUE TITLE
Test failures due to rowid columns

### ISSUE THREAD

**Issue by simonw** [simonw/datasette-edit-schema#33]
Tests are failing in CI but not on my laptop: https://github.com/simonw/datasette-edit-schema/runs/7157814881
```
>       assert table.columns_dict == expected_columns_dict
E       AssertionError: assert {'description...lass 'float'>} == {'description...<class 'int'>}
E         Omitting 2 identical items, use -vv to show
E         Right contains 1 more item:
E         {'rowid': <class 'int'>}
E         Full diff:
E         - {'description': <class 'str'>, 'name': <class 'float'>, 'rowid': <class 'int'>}
E         ?                                                       ------------------------
E         + {'description': <class 'str'>, 'name': <class 'float'>}
```

**Comment by simonw** [comment]
My laptop has an OLD version of `sqlite-utils`:
```
% pip freeze | grep sqlite
sqlite-fts4==1.0.1
sqlite-utils==3.0a0
```

**Comment by simonw** [comment]
It's because of a bug I fixed in `sqlite-utils` 3.10: https://sqlite-utils.datasette.io/en/stable/changelog.html#v3-10

> -   **Bug fix**: `table.transform()`, when run against a table without explicit primary keys, would incorrectly create a new version of the table with an explicit primary key column called `rowid`. ([#284](https://github.com/simonw/sqlite-utils/issues/284))
